keps/sig-node/1029-ephemeral-storage-quotas/README.md

```diff
@@ -840,65 +890,78 @@ If LocalStorageCapacityIsolationFSQuotaMonitoring is turned on but LocalStorageC
 
```

Comment:
In question `###### How can this feature be enabled / disabled in a live cluster?` you only mentioned kubelet with `LocalStorageCapacityIsolationFSQuotaMonitoring` flag. But earlier in the `#### Kubelet and API Server Skew:` section you're talking about both kubelet and apiserver. So I'd expect this question to explicitly point out both feature gates. The same applies to the `kep.yaml`, I've pointed out it's out of date, and there's a separate section describing feature gates and components.

Reply:
In `###### How can a rollout or rollback fail? Can it impact already running workloads?` you're saying that it won't affect already running workloads. What about the ones which enabled this functionality?

Reply:
In `###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?` you probably want to just say `No`, since you're not removing any fields, features, flags, or anything like that.